sp_api/base/helpers.py

```diff
@@ -3,10 +3,11 @@
 import base64
 import warnings
 import functools
+from urllib import parse
 
 
 def fill_query_params(query, *args):
-    return query.format(*args)
+    return query.format(*[parse.quote(arg, safe="") for arg in args])
```

Comment:
```suggestion
    return query.format(*[parse.quote(arg, safe='') for arg in args])
```
Let's keep the code style consistent